prow/pod-utils/decorate/podspec.go

```diff
@@ -398,8 +473,10 @@ func decorate(spec *kube.PodSpec, pj *kube.ProwJob, rawEnv map[string]string) er
 	})
 	spec.Volumes = append(spec.Volumes, logVolume, toolsVolume, gcsCredentialsVolume)
 
-	if willCloneRefs {
-		spec.Containers[0].WorkingDir = clone.PathForRefs(codeMountPath, refs[0])
+	if cloner != nil {
+		if r := pj.PrimaryRefs(); r != nil {
```

Comment:
When is it possible for `cloner` to be non-`nil` but there to be no primary refs?